src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1331,9 +1331,20 @@ bool isRefEscape
             // widest possible escape via writeable ref-like receiver or ref/out argument.
             uint escapeTo = scopeOfTheContainingExpression;
 
+            var isReadOnlyInvocation = symbol switch
+            {
+                MethodSymbol m => m.IsEffectivelyReadOnly,
+                // TODO: val escape checks should be skipped for property accesses when
```

Comment:
>TODO [](start = 19, length = 4)

Please don't leave bare todos in code. File a follow-up bug and link it here.

Reply:
Fixed